test/rest.py

```diff
@@ -68,9 +69,17 @@ def test_client(self):
         self.assertEqual(status, 200)
         self.assertEqual(body['merge_commit_sha'], u'fba3e13815f3d2a9dfbd2f89f1cf678dd58bb1f1')
 
-def suite():
-    """ returns all the testcases in this module """
-    return TestLoader().loadTestsFromTestCase(RestClientTest)
-
-if __name__ == '__main__':
-    main()
```

Comment:
what does that even mean? you can use `python setup.py test` `-F`  and `-f` flags.